<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-color: rgb(253, 251, 254);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_pi">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
}
QPushButton:pressed {
	border-style: outset;
	border-width: 2px;
	border-radius: 15px;
	border: none;
    background-color: rgb(227,225,228);
}</string>
    </property>
    <property name="text">
     <string>π</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sqrt">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
}
QPushButton:pressed {
	border-style: outset;
	border-width: 2px;
	border-radius: 15px;
	border: none;
    background-color: rgb(227,225,228);
}</string>
    </property>
    <property name="text">
     <string>√</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pow">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
}
QPushButton:pressed {
	border-style: outset;
	border-width: 2px;
	border-radius: 15px;
	border: none;
    background-color: rgb(227,225,228);
}</string>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mod">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>100</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
}
QPushButton:pressed {
	border-style: outset;
	border-width: 2px;
	border-radius: 15px;
	border: none;
    background-color: rgb(227,225,228);
}</string>
    </property>
    <property name="text">
     <string>mod</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
}
QPushButton:pressed {
	border-style: outset;
	border-width: 2px;
	border-radius: 15px;
	border: none;
    background-color: rgb(227,225,228);
}</string>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sin">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
}
QPushButton:pressed {
	border-style: outset;
	border-width: 2px;
	border-radius: 15px;
	border: none;
    background-color: rgb(227,225,228);
}</string>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_tan">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
}
QPushButton:pressed {
	border-style: outset;
	border-width: 2px;
	border-radius: 15px;
	border: none;
    background-color: rgb(227,225,228);
}</string>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ln">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>130</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
}
QPushButton:pressed {
	border-style: outset;
	border-width: 2px;
	border-radius: 15px;
	border: none;
    background-color: rgb(227,225,228);
}</string>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_asin">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>160</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
}
QPushButton:pressed {
	border-style: outset;
	border-width: 2px;
	border-radius: 15px;
	border: none;
    background-color: rgb(227,225,228);
}</string>
    </property>
    <property name="text">
     <string>asin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_acos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
}
QPushButton:pressed {
	border-style: outset;
	border-width: 2px;
	border-radius: 15px;
	border: none;
    background-color: rgb(227,225,228);
}</string>
    </property>
    <property name="text">
     <string>acos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_log">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>160</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
}
QPushButton:pressed {
	border-style: outset;
	border-width: 2px;
	border-radius: 15px;
	border: none;
    background-color: rgb(227,225,228);
}</string>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_atan">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>160</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
}
QPushButton:pressed {
	border-style: outset;
	border-width: 2px;
	border-radius: 15px;
	border: none;
    background-color: rgb(227,225,228);
}</string>
    </property>
    <property name="text">
     <string>atan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_open">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>240</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-style: outset;
	border-width: 2px;
	border-radius: 25px;
	background-color: rgb(223, 225, 248);
	border: none;
}

QPushButton:pressed {
    border-style: outset;
	border-width: 2px;
	border-radius: 15px;
	background-color: rgb(198, 202, 229);
	border: none;
}</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-style: outset;
	border-width: 2px;
	border-radius: 25px;
	background-color: rgb(251, 212, 243);
	border: none;
}
QPushButton:pressed {
    background-color: rgb(229, 193, 219);
	border-style: outset;
	border-width: 2px;
	border-radius: 15px;
	border: none;
}</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_div">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>240</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-style: outset;
	border-width: 2px;
	border-radius: 25px;
	background-color: rgb(223, 225, 248);
	border: none;
}

QPushButton:pressed {
    border-style: outset;
	border-width: 2px;
	border-radius: 15px;
	background-color: rgb(198, 202, 229);
	border: none;
}</string>
    </property>
    <property name="text">
     <string>÷</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_close">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>240</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-style: outset;
	border-width: 2px;
	border-radius: 25px;
	background-color: rgb(223, 225, 248);
	border: none;
}

QPushButton:pressed {
    border-style: outset;
	border-width: 2px;
	border-radius: 15px;
	background-color: rgb(198, 202, 229);
	border: none;
}</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mult">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>300</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-style: outset;
	border-width: 2px;
	border-radius: 25px;
	background-color: rgb(223, 225, 248);
	border: none;
}

QPushButton:pressed {
    border-style: outset;
	border-width: 2px;
	border-radius: 15px;
	background-color: rgb(198, 202, 229);
	border: none;
}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>360</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-style: outset;
    border-width: 2px;
    border-radius: 25px;
	background-color: rgb(245, 242, 249);
	border: none;
}
QPushButton:pressed {
   	border-style: outset;
    border-width: 2px;
    border-radius: 15px;
	background-color: rgb(220, 217, 224);
	border: none;
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>300</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-style: outset;
    border-width: 2px;
    border-radius: 25px;
	background-color: rgb(245, 242, 249);
	border: none;
}
QPushButton:pressed {
   	border-style: outset;
    border-width: 2px;
    border-radius: 15px;
	background-color: rgb(220, 217, 224);
	border: none;
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-style: outset;
    border-width: 2px;
    border-radius: 25px;
	background-color: rgb(245, 242, 249);
	border: none;
}
QPushButton:pressed {
   	border-style: outset;
    border-width: 2px;
    border-radius: 15px;
	background-color: rgb(220, 217, 224);
	border: none;
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_minus">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>360</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-style: outset;
	border-width: 2px;
	border-radius: 25px;
	background-color: rgb(223, 225, 248);
	border: none;
}

QPushButton:pressed {
    border-style: outset;
	border-width: 2px;
	border-radius: 15px;
	background-color: rgb(198, 202, 229);
	border: none;
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-style: outset;
    border-width: 2px;
    border-radius: 25px;
	background-color: rgb(245, 242, 249);
	border: none;
}
QPushButton:pressed {
   	border-style: outset;
    border-width: 2px;
    border-radius: 15px;
	background-color: rgb(220, 217, 224);
	border: none;
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>360</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-style: outset;
    border-width: 2px;
    border-radius: 25px;
	background-color: rgb(245, 242, 249);
	border: none;
}
QPushButton:pressed {
   	border-style: outset;
    border-width: 2px;
    border-radius: 15px;
	background-color: rgb(220, 217, 224);
	border: none;
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>300</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-style: outset;
    border-width: 2px;
    border-radius: 25px;
	background-color: rgb(245, 242, 249);
	border: none;
}
QPushButton:pressed {
   	border-style: outset;
    border-width: 2px;
    border-radius: 15px;
	background-color: rgb(220, 217, 224);
	border: none;
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>420</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-style: outset;
	border-width: 2px;
	border-radius: 25px;
	background-color: rgb(223, 225, 248);
	border: none;
}

QPushButton:pressed {
    border-style: outset;
	border-width: 2px;
	border-radius: 15px;
	background-color: rgb(198, 202, 229);
	border: none;
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dot">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>480</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-style: outset;
    border-width: 2px;
    border-radius: 25px;
	background-color: rgb(245, 242, 249);
	border: none;
}
QPushButton:pressed {
   	border-style: outset;
    border-width: 2px;
    border-radius: 15px;
	background-color: rgb(220, 217, 224);
	border: none;
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>420</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-style: outset;
    border-width: 2px;
    border-radius: 25px;
	background-color: rgb(245, 242, 249);
	border: none;
}
QPushButton:pressed {
   	border-style: outset;
    border-width: 2px;
    border-radius: 15px;
	background-color: rgb(220, 217, 224);
	border: none;
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-style: outset;
    border-width: 2px;
    border-radius: 25px;
	background-color: rgb(245, 242, 249);
	border: none;
}
QPushButton:pressed {
   	border-style: outset;
    border-width: 2px;
    border-radius: 15px;
	background-color: rgb(220, 217, 224);
	border: none;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_equal">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>480</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-style: outset;
	border-width: 2px;
	border-radius: 25px;
	background-color: rgb(223, 225, 248);
	border: none;
}

QPushButton:pressed {
    border-style: outset;
	border-width: 2px;
	border-radius: 15px;
	background-color: rgb(198, 202, 229);
	border: none;
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-style: outset;
    border-width: 2px;
    border-radius: 25px;
	background-color: rgb(245, 242, 249);
	border: none;
}
QPushButton:pressed {
   	border-style: outset;
    border-width: 2px;
    border-radius: 15px;
	background-color: rgb(220, 217, 224);
	border: none;
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_erase">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>480</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-style: outset;
    border-width: 2px;
    border-radius: 25px;
	background-color: rgb(245, 242, 249);
	border: none;
}
QPushButton:pressed {
   	border-style: outset;
    border-width: 2px;
    border-radius: 15px;
	background-color: rgb(220, 217, 224);
	border: none;
}</string>
    </property>
    <property name="text">
     <string>⌫</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>420</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-style: outset;
    border-width: 2px;
    border-radius: 25px;
	background-color: rgb(245, 242, 249);
	border: none;
}
QPushButton:pressed {
   	border-style: outset;
    border-width: 2px;
    border-radius: 15px;
	background-color: rgb(220, 217, 224);
	border: none;
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_x">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-style: outset;
	border-width: 2px;
	border-radius: 15px;
	border: none;
	background-color: rgb(245, 242, 249);
}
QPushButton:pressed {
	border-style: outset;
	border-width: 2px;
	border-radius: 15px;
	border: none;
    background-color: rgb(227,225,228);
}</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="x_value">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>200</y>
      <width>131</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	border-style: outset;
	border-width: 2px;
	border-radius: 15px;
	border: none;
    background-color: rgb(245, 242, 249);
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="result_show">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>270</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
border-style: outset;
	border-width: 2px;
	border-radius: 25px;
	border: none;
background-color : rgb(229,229,241);
}</string>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
